### PR TITLE
Improvement/allow override of SDK default autofocus behaviour

### PR DESCRIPTION
**Pull request by kopijunkie**
# Problem

The SDK default UX behaviour to programatically set focus on each screen's title is not desirable on some integrations as this UX causes the host site/app to focus to where the SDK screen title element is on loading and initialising the SDK so end users miss content, form inputs outside of the SDK that they should see, fill in first unless they know to scroll back up to see what it is.  

See discussion in issue #1539 

# Solution

Allow integrators to optionally set a `autoFocusOnInitialScreenTitle: false` option when initialising the SDK. Based on that option, override `pageTitle.focus()` behaviour for the integrator's initial screen only whether it's the Welcome or Doc Selection screen, etc.

## Checklist

_put `n/a` if item is not relevant to PR changes_

- [x] Has the CHANGELOG been updated?
- [x] Has the README been updated?
- [ ] Has the CONTRIBUTING doc been updated?
- [ ] Has the RELEASE_GUIDELINES been updated?
- [ ] Has the TESTING_GUIDELINES been updated?
- [ ] Has the MIGRATION doc been updated for any MAJOR breaking changes?
- [ ] Has the MIGRATION doc been updated for any MINOR breaking changes, including any translation strings or keys changes?
- [ ] Have any new automated tests been implemented or the existing ones changed?
- [ ] Have any new manual tests been written down or the existing ones changed?
- [ ] Have any new strings been translated or the existing ones changed?

### CODE REVIEW

File: src/components/Welcome/index.tsx [hunk 1]
```diff
@@ -79,7 +81,11 @@ const Welcome: FunctionComponent<StepComponentBaseProps> = ({
 
   return (
     <ScreenLayout actions={actions} className={style.container}>
-      <PageTitle title={welcomeTitle} subTitle={welcomeSubTitle} />
+      <PageTitle
+        title={welcomeTitle}
+        subTitle={welcomeSubTitle}
+        shouldAutoFocus={isFirstScreen && autoFocusOnInitialScreenTitle}
```
Comment (rabelloo) [inline src/components/Welcome/index.tsx:87]: I see this prop being passed in several places, but I don't see a change to `PageTitle` itself to do anything with the prop.
Did you forget a commit, or am I missing something?

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,12 @@ This change log file is based on best practices from [Keep a Changelog](http://k
 This project adheres to [Semantic Versioning](http://semver.org/). Breaking changes result in a different MAJOR version. UI changes that might break customizations on top of the SDK will be treated as breaking changes too.
 This project adheres to the Node [default version scheme](https://docs.npmjs.com/misc/semver).
 
+## [next-next-version]
+
+### Changed
+
+- Public: Added `autoFocusOnInitialScreenTitle` SDK configuration option for integrators to override SDK auto focusing on the SDK's initial screen title on loading. This default behaviour might not be desireable for some host apps or sites as this could cause the browser to auto scroll to the SDK if there is content that the end user should see first.
```
Comment (rabelloo) [inline CHANGELOG.md:13]: 🤓  [discussion]
If it's just the scrolling that is the issue, and focusing the title is always desirable for accessibility, we could simply `preventScroll` and keep the `focus`, i.e.
`containerRef.current?.focus({ preventScroll: true })`

From the linked issue I get the impression that it's not just scrolling though, they also want to keep focus on a different "out-of-SDK" textarea even after the initial SDK screen loads and renders.
If that's the case, we should probably document accordingly, lest we introduce a regression for that use case.

File: src/components/Welcome/index.tsx (hunk 1 above)
Comment (kopijunkie) [reply src/components/Welcome/index.tsx:87]: This prop already exists in [`PageTitle`](https://github.com/onfido/onfido-sdk-ui/blob/master/src/components/PageTitle/index.tsx#L23) as it was introduced in an earlier PR #1560 hence why there isn't any change to `PageTitle` in this PR.

File: CHANGELOG.md (hunk 2 above)
Comment (kopijunkie) [reply CHANGELOG.md:13]: That's true. 😅  I've forgotten about the "out-of-SDK" textarea in the discussion when writing this note. Which I think is also the main issue for a particular client that is waiting on this. Will rephrase/edit this CHANGELOG entry.

File: src/components/Welcome/index.tsx (hunk 1 above)
Comment (rabelloo) [reply src/components/Welcome/index.tsx:87]: I see, I was looking at the wrong branch, my mistake

**Review by rabelloo — APPROVED** [review]

**Review by ethan605 — APPROVED** [review]
LGTM!

File: src/components/ProofOfAddress/PoAIntro/index.js [hunk 3]
```diff
@@ -8,37 +8,51 @@ import { POA_INTRO_LOCALES_MAPPING } from '~utils/localesMapping'
...
-      ))}
+const PoAIntro = ({
+  country,
+  translate,
+  parseTranslatedTags,
+  nextStep,
+  steps,
+  autoFocusOnInitialScreenTitle,
+}) => {
+  const isFirstScreen = steps[0].type === 'face'
```
Comment (stefaniacardenas) [inline src/components/ProofOfAddress/PoAIntro/index.js:19]: Why do we assume that face is the first screen on the PoA flow?

Comment (kopijunkie) [reply src/components/ProofOfAddress/PoAIntro/index.js:19]: Ah, good spot. I meant for it to be `'poa'`, not `'face'` in this case.

**Review by stefaniacardenas — APPROVED** [review]

**Pull request merged**